lib/challenge.js

```diff
@@ -48,7 +62,19 @@ var pet_owner;
   // Every pet can:
   // - make noise (each pet makes its own unique noise, as specified by `noise`.
 
-var some_pet;
+var some_pet {
+
+name: "John Doe",
+species: "Dob",
+breed: "Chia",
+noise: [
+{
+  noise1: "woof",
+  noise2: "WOOOOOF",
+  noise3: "meow"
+}
+],
+ ;}
```

Comment:
Semicolons have no place inside an object literal. Without fail, they will result in a syntax error being thrown. There are two types of curly braces: the kind the follow functions, loops, and `if`f, which contain **statements**, and the kind that contain **key-value pairs**.

Statements are actions. They lead to things being done. The semicolon (`;`) is the statement terminator. It's like a period at the end of a sentence.

Key-value pairs are essentially property assignments. Their syntax is as follows:

`keyName: value`

That is, an identifier (or string), a colon (`:`), and a value expression. Multiple key-value pairs must be separated using commas (`,`). While directly inside an object literal, only key-value pairs are syntactically correct.

Back to the question, it wanted you to place a method on an object literal to log a pet-like sound. This object literal doesn't have a method.

1/2